core/geometry/src/test/geometry3d/FrameBuilder.test.ts

```diff
@@ -218,13 +219,15 @@ describe("FrameBuilder", () => {
         AxisScaleSelect.NonUniformRangeContainment]) {
         const localToWorld = FrameBuilder.createLocalToWorldTransformInRange(range, select, 0, 0, 0, 2.0);
         if (ck.testPointer(localToWorld)) {
-          MatrixTests.checkProperties(ck,
+          MatrixTests.checkProperties(
+            ck,
             localToWorld.matrix,
             select === AxisScaleSelect.Unit,  // unit axes in range are identity
             select === AxisScaleSelect.Unit,  // and of course unitPerpendicular
             select === AxisScaleSelect.Unit,  // and of course rigid.
             true, // always invertible
-            true);  // always diagonal
+            true
+          );  // always diagonal
```

Comment:
This comment ("always diagonal") belongs to the last arg, so I'd move it up a line (to 229).

Reply:
will fix